src/map/lua/lua_baseentity.cpp

```diff
@@ -14422,6 +14422,77 @@ bool CLuaBaseEntity::deleteRaisedChocobo()
     return true;
 }
 
+void CLuaBaseEntity::setMannequinPose(uint16 itemID, uint8 race, uint8 pose)
+{
+    TracyZoneScoped;
+    // Find the item in the player inventory and update the extra values
+    auto*  PChar = dynamic_cast<CCharEntity*>(m_PBaseEntity);
```

Comment:
If you dynamic cast, you should be checking the result immediately after, like you did below with the mannequin